### PR TITLE
i18n 1.1 changed fallbacks to exclude default locale.

### PR DESCRIPTION
**Pull request by pgwillia**
But that may break your application.

Please check your Rails app for 'config.i18n.fallbacks = true'.
If you're using I18n (>= 1.1.0) and Rails (< 5.2.2), this should be
'config.i18n.fallbacks = [I18n.default_locale]'.
If not, fallbacks will be broken in your app by I18n 1.1.x.

For more info see:
https://github.com/svenfuchs/i18n/releases/tag/v1.1.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and releases in Jupiter project adheres to [Semantic Versioning](http://semver.o
 
 ## [Unreleased]
 
+### Changed
+- i18n fallback to english (configuration change) [PR#1058](https://github.com/ualbertalib/discovery/issues/1558)
```
Comment (weiweishi) [inline CHANGELOG.md]: Why we are referring a discovery issue here?

Comment (pgwillia) [reply CHANGELOG.md]: Copy paste error. Should be https://github.com/ualbertalib/jupiter/pull/1058

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,10 +6,15 @@ and releases in Jupiter project adheres to [Semantic Versioning](http://semver.o
...
 
+### Changed
+- i18n fallback to english (configuration change) [PR#1058](https://github.com/ualbertalib/jupiter/pull/1058)
+
 ## [1.2.14] - 2019-04-15
 
 ### Added
 - regression tests for downloading restricted items from search results [PR#1070](https://github.com/ualbertalib/jupiter/pull/1070)
+- Added danger gem to project [#988](https://github.com/ualbertalib/jupiter/issues/998)
+- Added rack-attack for rate limiting [#954](https://github.com/ualbertalib/jupiter/issues/954)
```
Comment (pgwillia) [inline CHANGELOG.md]: These ended up in the wrong spot in the Changelog.  Been trying to fix them.

Comment (weiweishi) [reply CHANGELOG.md]: Thanks! I might have not caught them during review. Sorry for the trouble.

**Pull request merged**